src/commands/init.js

```diff
@@ -248,22 +259,21 @@ function interactiveMenu(directory, list) {
     });
 }
 
-function checkFolder(force = false, directoryName = '') {
+function checkFolder(force = false, directory = '') {
     return new Promise((resolve) => {
-        const directoryPath = path.join(process.cwd(), directoryName);
+        const directoryPath = getAbsolutePath(directory || process.cwd());
```

Comment:
I think it would be better to change the signature for `checkFolder` to `(force, directoryName, directory)` and use `directory` as the second  argument to `getAbsolutePath`. That means that https://github.com/rocjs/roc/pull/109/files#diff-03189079b1b3d77ef73e9e3fdb324d6dR47 would be similar to before.


What do you think?